examples/elk/converge/elk.hcl

```diff
@@ -0,0 +1,114 @@
+param "user-name" {
+  default = "vagrant"
+}
+
+module "packages.hcl" "packages" {}
+
+module "docker.hcl" "docker" {
+  params = {
+    user-name = "{{param `user-name`}}"
+  }
+  depends = ["module.packages/task.epel-install"]
```

Comment:
depending on a task inside another module is unsupported. It works, but it probably won't in the future. I thought we had disallowed it, actually. Depending on `module.packages` gets you the same result.